src/app/pages/FrontPage/index.jsx

```diff
@@ -132,8 +134,11 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
     />
   );
 
+  const { isAmp } = useContext(RequestContext);
+
   return (
     <>
+      {!isAmp && <CanonicalAdBootstrapJs />}
```

Comment:
This bootstrap JS will then be run all all pages on all environments?

Reply:
Just front page, `FrontPage/index.jsx`

Reply:
Aha, sorry - the setup code has changed to not need to call the bootstrap function any more, so this now just sets up an object, I was still under the impression this was going to be calling the bootstrap function - my mistake

Reply:
Why do we import it here instead of in the `CanonicalAd` container?

Reply:
> Why do we import it here instead of in the CanonicalAd container?

This code looks like we only need to call it once, for instance, and if we have the mpu Ad in `CanonicalAd` container, it would lead to us calling this function twice.